packages/codecs/ilp/src/tsconfig.build.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../../tsconfig.json",
+  "compilerOptions": {
+    "composite": true,
+    "baseUrl": ".",
+    "outDir": "../build"
```

Comment:
with the `tsconfig.build.json` non default named `tsconfig.json`, the editors or language service don't seem to get confused and try and resolve the `@interledger/*` absolute imports from this directory ... 

but with the `include` and the `baseUrl` set to `.` of course, the build/ output matches the src/

Reply:
Nice!